chord_metadata_service/cleanup/tests/test_cleanup.py

```diff
@@ -118,15 +156,17 @@ async def test_cleanup_basic(self):
 
         # 1 metadata object +
         # 2 biosamples +
-        # 1 procedure +
         # 0 linked phenotypic feature (removed via cascade with v2.17.0 database changes) +
         # 1 unlinked phenotypic feature (pretend left-over from pre v2.17.0 or created manually) +
         # 1 individual +
         # 0 experiment results +
         # 0 instruments +
-        # 1 resource
-        # = 7 objects total
-        self.assertEqual(await run_all_cleanup(), 7)
+        # 1 resource +
+        # 1 interpretation +
+        # 1 diagnosis
+        # 1 genomic interpretation
```

Comment:
group these in the comment with the other phenopacket-related cleanup stuff, then experiments/related comes after